docs/install/update-pro.md

```diff
@@ -8,85 +8,116 @@ This document provides instructions how you can upgrade from Percona Server for
+            ```            
 
-        ```ini title="/etc/yum.repos.d/psmdb-pro.repo"
-        [psmdb-7.0-pro]
-        name=PSMDB_7.0_PRO
-        baseurl=http://repo.percona.com/private/[TOKENID]-[TOKEN]/psmdb-70-pro/yum/main/$releasever/RPMS/x86_64
-        enabled=1
-        gpgkey = https://repo.percona.com/yum/PERCONA-PACKAGING-KEY
-        ```
+        === ":octicons-file-code-24: Configuration file"            
 
-3. Install Percona Server for MongoDB Pro packages
+            Create the `/root/.percona-private-repos.config` configuration file with the following content:            
 
-    === ":material-debian: On Debian and Ubuntu"
+            ```ini title="/root/.percona-private-repos.config"
+            [psmdb-70-pro]
+            USER_NAME=<Your Customer ID>
+            REPO_TOKEN=<Your PRO repository token>
+            ```            
+   
+    4. Install Percona Server for MongoDB Pro packages
 
         ```{.bash .data-prompt="$"}
         $ sudo apt install -y percona-server-mongodb-pro
         ```
 
-    === ":material-redhat: On RHEL 8+ and derivatives"
+    5. Start the server            
 
         ```{.bash .data-prompt="$"}
-        $ sudo yum install -y percona-server-mongodb-pro
+        $ sudo systemctl start mongod
+        ``` 
+
+=== ":material-redhat: On RHEL and derivatives"
+
+    1. Stop the `mongod` service    
+
+        ```{.bash data-prompt="$"}
+        $ sudo systemctl stop mongod
         ```
 
-    === ":material-redhat:  On RHEL 7 and derivatives"
+    2. [Install `percona-release` :octicons-link-external-16:](https://docs.percona.com/percona-software-repositories/installing.html#__tabbed_1_2). If you have installed it before, [upgrade :octicons-link-external-16:](https://docs.percona.com/percona-software-repositories/updating.html#__tabbed_1_2) it to the latest version.
+    3. Enable the repository. Choose your preferable method:            
 
-        1. Back up the `/etc/mongod.conf` configuration file
-       
-            ```{.bash .data-prompt="$"}
-            $ sudo cp /etc/mongod.conf /etc/mongod.conf.bkp
-            ```
+        === ":material-console: Command line"    
 
-        2. Remove basic packages of Percona Server for MongoDB 
+            Run the following command and pass your credentials to the Pro repository:            
 
             ```{.bash .data-prompt="$"}
-            $ sudo yum remove percona-server-mongodb*
-            ```
+            $ sudo percona-release enable psmdb-70-pro release --user_name=<Your Customer ID> --repo_token=<Your PRO repository token>
+            ```             
+
+        === ":octicons-file-code-24: Configuration file"            
+
+            Create the `/root/.percona-private-repos.config` configuration file with the following content:            
+
+            ```ini title="/root/.percona-private-repos.config"
+            [psmdb-70-pro]
+            USER_NAME=<Your Customer ID>
+            REPO_TOKEN=<Your PRO repository token>
```

Comment:
@radoslawszulgo, do you think it is feasible to remove this option? According to @sandraromanchenko we need to run the command percona-release enable psmdb-70-pro anyway. So the option to define creds in the config file seems to bring little value

Reply:
Yes, that's essential piece. Why do we wanted to remove it?